lisa/operating_system.py

```diff
@@ -554,33 +554,21 @@ def _get_os_version(self) -> OsVersion:
 
 
 class Redhat(Fedora):
-    # pattern for expired RHUI client certificate issue
-    __rhui_error_pattern = re.compile(
-        r"([\w\W]*?)SSL peer rejected your certificate as expired.*", re.MULTILINE
-    )
-
     @classmethod
     def name_pattern(cls) -> Pattern[str]:
         return re.compile("^rhel|Red|AlmaLinux|Scientific|acronis|Actifio$")
 
     def _initialize_package_installation(self) -> None:
-        # older images cost much longer time when update packages
-        # smaller sizes cost much longer time when update packages, e.g.
-        #  Basic_A1, Standard_A5, Standard_A1_v2, Standard_D1
-        # redhat rhel 7-lvm 7.7.2019102813 Basic_A1 cost 2371.568 seconds
-        # redhat rhel 8.1 8.1.2020020415 Basic_A0 cost 2409.116 seconds
-        cmd_result = self._node.execute("yum -y update", sudo=True, timeout=3600)
+        cmd_result = self._node.execute("yum makecache", sudo=True)
```

Comment:
The original code can still keep, we can add a method like `update_packages`. If we need to update existing package, it can be called.